boards/arm64/fvp_base_revc_2xaemv8a/board.cmake

```diff
@@ -29,4 +29,16 @@ set(ARMFVP_FLAGS
   -C bp.vis.rate_limit-enable=0
   -C gic_distributor.ARE-fixed-to-one=1
   -C gic_distributor.ITS-device-bits=16
+  -C cache_state_modelled=0
   )
+
+if(CONFIG_BUILD_WITH_TFA)
+  set(TFA_PLAT "fvp")
+  if(CONFIG_TFA_MAKE_BUILD_TYPE_DEBUG)
+    set(FVP_SECURE_FLASH_FILE ${TFA_BINARY_DIR}/fvp/debug/bl1.bin)
+    set(FVP_FLASH_FILE ${TFA_BINARY_DIR}/fvp/debug/fip.bin)
+  else()
+    set(FVP_SECURE_FLASH_FILE ${TFA_BINARY_DIR}/fvp/release/bl1.bin)
+    set(FVP_FLASH_FILE ${TFA_BINARY_DIR}/fvp/release/fip.bin)
+  endif()
+endif()
```

Comment:
no new line at the end of file

Reply:
I didn't see any CMake file that need an extra new line at the end of the file. Do we really need it?